owasp-suppressions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+    <suppress>
+        <notes><![CDATA[
+   file name: avro-1.9.2.jar
```

Comment:
Please include an explanation of why we're suppressing this vulnerability (e.g. "According to https://nvd.nist.gov/vuln/detail/cve-2021-43045 , CVE only impacts .NET SDK")